powerfulseal/policy/ps-schema.yaml

```diff
@@ -216,6 +224,14 @@ definitions:
               - "$ref": "#/definitions/filterDayTime"
               - "$ref": "#/definitions/filterRandomSample"
               - "$ref": "#/definitions/filterProbability"
+          retries:
+            type: object
+            description: >
+              An object of retry criteria to rerun set actions
+            items:
```

Comment:
Same applies here - there should no longer be `items` here, like [here](https://github.com/powerfulseal/powerfulseal/blob/570d8b0921c35ccf4bfe0ee4a465d93d414290c8/powerfulseal/policy/ps-schema.yaml#L305-L309).

Let's also add this to the kitchen sink validation test in [here](https://github.com/powerfulseal/powerfulseal/blob/570d8b0921c35ccf4bfe0ee4a465d93d414290c8/tests/policy/example_config.yml). This way it will get validated in the CI, and we'll know for sure.